cancellation/README.md

```diff
@@ -0,0 +1,16 @@
+# VSCode Language Server - Protocol Module
+
+[![NPM Version](https://img.shields.io/npm/v/vscode-languageserver-protocol.svg)](https://npmjs.org/package/vscode-languageclient)
```

Comment:
The need to point to the corresponding npm package

Reply:
sure, but I dont have one yet.